RecoTracker/IterativeTracking/python/DetachedTripletStep_cff.py

```diff
@@ -214,6 +214,29 @@
 detachedTripletStepTrackCandidates.TrajectoryCleaner = 'detachedTripletStepTrajectoryCleanerBySharedHits'
 trackingLowPU.toModify(detachedTripletStepTrajectoryCleanerBySharedHits, fractionShared = 0.19)
 
+from Configuration.ProcessModifiers.trackingMkFitDetachedTripletStep_cff import trackingMkFitDetachedTripletStep
+import RecoTracker.MkFit.mkFitSeedConverter_cfi as mkFitSeedConverter_cfi
+import RecoTracker.MkFit.mkFitIterationConfigESProducer_cfi as mkFitIterationConfigESProducer_cfi
+import RecoTracker.MkFit.mkFitProducer_cfi as mkFitProducer_cfi
+import RecoTracker.MkFit.mkFitOutputConverter_cfi as mkFitOutputConverter_cfi
+detachedTripletStepTrackCandidatesMkFitSeeds = mkFitSeedConverter_cfi.mkFitSeedConverter.clone(
+    seeds = 'detachedTripletStepSeeds',
+)
+detachedTripletStepTrackCandidatesMkFitConfig = mkFitIterationConfigESProducer_cfi.mkFitIterationConfigESProducer.clone(
+    ComponentName = 'detachedTripletStepTrackCandidatesMkFitConfig',
+    config = 'RecoTracker/MkFit/data/mkfit-phase1-detachedTripletStep.json',
+)
+detachedTripletStepTrackCandidatesMkFit = mkFitProducer_cfi.mkFitProducer.clone(
+    seeds = 'detachedTripletStepTrackCandidatesMkFitSeeds',
+    config = ('', 'detachedTripletStepTrackCandidatesMkFitConfig'),
+    clustersToSkip = 'detachedTripletStepClusters',
+)
+trackingMkFitDetachedTripletStep.toReplaceWith(detachedTripletStepTrackCandidates, mkFitOutputConverter_cfi.mkFitOutputConverter.clone(
+    seeds = 'detachedTripletStepSeeds',
+    mkfitSeeds = 'detachedTripletStepTrackCandidatesMkFitSeeds',
+    tracks = 'detachedTripletStepTrackCandidatesMkFit',
+))
+
```

Comment:
we will need to update the FastSim as well ?

Reply:
Why? FastSim pattern recognition is something very different anyway.